main/models/room.py

```diff
@@ -0,0 +1,24 @@
+
+class Room(object):
+    """Room is the parent and Office and Living_space inherit from it"""
+
+    def __init__(self, connection):
+        self.connect = connection
+        #self.name = name
```

Comment:
**Code Quality: Nitpick**

Remove this line. Also not sure what `connection` is doing here.